src/modules/mavlink/mavlink_params.c

```diff
@@ -155,3 +155,13 @@ PARAM_DEFINE_INT32(MAV_HB_FORW_EN, 1);
  * @max 250
  */
 PARAM_DEFINE_INT32(MAV_RADIO_TOUT, 5);
+
+/**
+ * Clear rally points with approaches on startup
```

Comment:
It's not clear to me from this param description if it means that "only rally points with approaches" or "all rally points AND their corresponding approaches" are cleared or